test/test_functions

```diff
@@ -1383,17 +1383,18 @@ set_valgrind_options() {
 # @param workspace  absolute path to the directory where the benchmark will put the files in
 setup_benchmark_environment() {
   local workspace="$1"
-  mkdir -p /cvmfs/$FQRN
+  local me=$(whoami)
+  sudo sh -c "mkdir -p /cvmfs/$FQRN && chown $me /cvmfs/$FQRN"
   mkdir -p "$CVMFS_OPT_OUTPUT_DIR/$FQRN"
   mkdir -p $workspace/cache
-
-  # cleaning the cache
-  clean_benchmark_cache
-
+  
+  # make sure the user can execute /bin/fusermount, write into /dev/fuse and write into /cvmfs/<repo>
+  sudo sh -c "usermod -a -G fuse $me"
```

Comment:
I think it's better to change the mode of /bin/fusermount and /dev/fuse to be world accessible.  You then don't have to care about the group membership of sftnight.